pkg/master/thirdparty_controller_test.go

```diff
@@ -90,8 +116,8 @@ func TestSyncAPIs(t *testing.T) {
 				Items: resourcesNamed("foo.example.com"),
 			},
 			apis: []string{
-				"/apis/example.com",
-				"/apis/example.com/v1",
+				"/apis/example.com/foos",
```

Comment:
Maybe I misunderstand the test, but I see it checking below that paths like this one (sans version) are removed. But such paths should not be created in the first place, so I'm confused?